front/app/containers/Admin/invitations/index.tsx

```diff
@@ -28,16 +32,25 @@ const InvitationsPage = (props: WrappedComponentProps) => {
     subtitle: props.intl.formatMessage(messages.invitationSubtitle),
   };
 
+  const goBack = () => {
+    clHistory.goBack();
```

Comment:
If I go to this page http://localhost:3000/en/admin/invitations, "Go back" doesn't work. Shouldn't it just lead to the users page?

Reply:
What happens when you click it? For my case, in chrome it works but it takes you back to the link you were on before. I think if we use `Go back`, that is how it should behave. Breadcrumbs are probably better but we can discuss that separately from the design

Reply:
Sorry for the confusion, I meant when I go directly to this link (e.g., clicking it here).

Reply:
A good solution for this case is to hide the link for those cases. Let me do that